### PR TITLE
feat: icon slots for single slot controls

### PR DESCRIPTION
**Pull request by AdamJaggard**
Replaces the single named slot in controls with a generic one called "icon". Affects Airplay, Seek forwards and backwards buttons.

This is a breaking change.

### CODE REVIEW

**Review by luwes — APPROVED** [review]

**Comment by luwes** [comment]
> This is a breaking change.

### Remember to add to the upgrade guide 

https://github.com/muxinc/media-chrome/blob/main/docs/src/pages/en/migration/from-0.x-to-1.0.md

**Comment by gkatsev** [comment]
And add `BREAKING CHANGE: ...` in the footer of the commit when merging.

File: docs/src/pages/docs/en/migration/from-0.x-to-1.0.md [hunk 1]
```diff
@@ -342,3 +345,21 @@ Also, for any custom themes you may have created, as a result of moving from `ke
   </template>
 </template>
 ```
+
+## Using the new `icon` slot
+
+All components with a single icon now expose a slot called `icon` instead of a different named slot for each component. e.g:
```
Comment (cjpillsbury) [inline docs/src/pages/docs/en/migration/from-0.x-to-1.0.md:351]: suggestion(non-blocking): Consider adding a simple list here of all of the single element slot renames, similar to https://github.com/muxinc/media-chrome/blob/main/docs/src/pages/docs/en/migration/from-0.x-to-1.0.md?plain=1#L165-L169)

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
This looks great overall. A couple of suggestions on the docs side plus:
issue(blocking): rename `media-loading-indicator` icon slot for consistency.

File: docs/src/pages/docs/en/migration/from-0.x-to-1.0.md [hunk 2]
```diff
@@ -342,3 +345,27 @@ Also, for any custom themes you may have created, as a result of moving from `ke
...
+```
+
+**After**
+```html
+<media-airplay-button>
+  <svg slot="icon"></svg>
+</media-airplay-button>
+```
+
+**Affected components**
```
Comment (cjpillsbury) [inline docs/src/pages/docs/en/migration/from-0.x-to-1.0.md:367]: 👍

**Review by cjpillsbury — APPROVED** [review]
🥳

**Pull request merged**